### PR TITLE
cmd ref: remove unnecessary indices?

### LINKED ISSUE #727: docs: remove unnecessary index pages that serve as a table of contents
**Blocked by #738**

E.g. https://dvc.org/doc/user-guide/contributing

It'll be very painful to support them. We will need to keep titles up to date, list of them, snippets, etc, etc.

Let's not use pages like this at all or come with a way to generate them automatically.

cc @dashohoxha

### PR DESCRIPTION
**Pull request by jorgeorpinel**
~~Blocked by #891~~

Closes #727

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
**Blocked by #891**

Closes #727

**Comment by jorgeorpinel** [comment]
(Please do not review yet.)

File: public/static/docs/tutorials/index.md [hunk 1]
```diff
@@ -1,18 +0,0 @@
-# Tutorials
```
Comment (shcheklein) [inline public/static/docs/tutorials/index.md]: tbh: some kind of an index is useful in this case (and other cases?) - when give a direct link to tutorial it feels strange that it opens the same page as the first section? what do you think, @jorgeorpinel ?

Comment (jorgeorpinel) [reply public/static/docs/tutorials/index.md]: This is the only index I could find that definitely seemed unnecessary to me at this point, actually. Please see discussion in https://github.com/iterative/dvc.org/issues/727#issuecomment-571407120 (and the next comment too).

Comment (jorgeorpinel) [reply public/static/docs/tutorials/index.md]: To summarize, I agree that

> it feels strange that it opens the same page as the first section

But I also don't want to maintain these bullet lists.

Maybe just strip down these indices to only a single paragraph and a sentence like "Please chose a tutorial in the left navigation side bar or click Next to jump into the first category"?

Comment (shcheklein) [reply public/static/docs/tutorials/index.md]: Yep, we can strip the description to the essence of it. A paragraph or two. It does not hurt to mention that there are different kinds of tutorials though. It def tedious to maintain an exact description.

Comment (jorgeorpinel) [reply public/static/docs/tutorials/index.md]: OK I've stripped down the indices for tutorials (restored), use-cases, and user-guide 👯

So... #891 was only really for the contribution guide 😋

**Comment by jorgeorpinel** [comment]
> So... #891 was only really for the contribution guide 😋 (from https://github.com/iterative/dvc.org/pull/903#discussion_r364350355)

Actually, on this @shcheklein what about my https://github.com/iterative/dvc.org/issues/727#issuecomment-571407120 about cmd ref indices?

> - https://dvc.org/doc/command-reference/cache,
> - https://dvc.org/doc/command-reference/metrics,
> - https://dvc.org/doc/command-reference/pipeline, and
> - https://dvc.org/doc/command-reference/remote
>
> have useful, unique content but I'm not sure the cmd ref is the best place for those explanations. Should we take the opportunity to move them into a new **user-guide: "Basic Concepts"** (#550)?

**Comment by jorgeorpinel** [comment]
This would mean that the first subcommand opens by default. For cache -> cache/dir (good), for metrics and remote -> */add, and for pipeline -> pipeline/list

**Comment by shcheklein** [comment]
@jorgeorpinel hmm ... I think they are valuable. Not matter what, but `dvc list -h` is a legit command by itself and should be described. Or in the future we can decide to make `dvc remote` an alias of `dvc remote list`?

File: public/static/docs/tutorials/index.md (hunk 1 above)
Comment (shcheklein) [reply public/static/docs/tutorials/index.md]: Cool! 😎

**Comment by jorgeorpinel** [comment]
Yes, they have valuable content explaining what the concepts are (cache, metrics, pipeline, remote). My point is that such content probably doesn't belong in the command reference and we can move it to a new **user-guide: "Basic Concepts"** (per #550) – and once that is done, it won't make sense to keep these indices.

Should they just be a short list of links to their subcommands? Or remove completely and let the first one load. I think at least for `dvc cache` since it only has `dir` inside, we should just remove the index.

`dvc list`? That's not released yet. Nor related to the commands I mentioned above ^ :suspect:

**Comment by shcheklein** [comment]
@jorgeorpinel `dvc remote`, `dvc cache`, etc are valid commands. So I think they belong to the command reference and should have more or less the same structure.

**Comment by shcheklein** [comment]
> dvc list? That's not released yet. Nor related to the commands I mentioned above ^ :suspect:

:) sorry, that was a mistake - I've updated the comment

**Comment by jorgeorpinel** [comment]
> dvc remote, dvc cache, etc are valid commands

Ah yes good point. Didn't think about that. So we definitely need the indices. OK! This PR is now officially reviewable!

**Comment by jorgeorpinel** [comment]
p.s. but please note that I've extracted the discussion about moving concepts explanations out of the cmd ref to https://github.com/iterative/dvc.org/issues/550#issuecomment-572223386.

**Pull request merged**